Chutzpah/TestRunner.cs

```diff
@@ -296,7 +296,19 @@ private bool PerformBatchCompile(ITestMethodRunnerCallback callback, IEnumerable
                                 "Launching test harness '{0}' for file '{1}' in a browser",
                                 testContext.TestHarnessPath,
                                 testContext.FirstInputTestFile);
-                            process.LaunchFileInBrowser(testContext.TestHarnessPath, options.BrowserName);
+
+                            // Allow override from command line.
+                            var browserArgs = testContext.TestFileSettings.BrowserArguments;
+                            if (!string.IsNullOrWhiteSpace(options.BrowserArgs))
+                            {
+                                var path = BrowserPathHelper.GetBrowserPath(options.BrowserName);
+                                browserArgs = new Dictionary<string, string>
+                                {
+                                    { Path.GetFileNameWithoutExtension(path), options.BrowserArgs }

```

Comment:
Why do you need to do ```Path.GetFileNameWithoutExtension(path)``` instead of just using options.BrowserName? Are you trying to account for case of BrowserName being null?